boilerplate/babel.config.js

```diff
@@ -1,4 +1,13 @@
 const plugins = [
+  [
+    "babel-plugin-root-import",
+    {
+      root: __dirname,
+      rootPathPrefix: "~/",
```

Comment:
I believe from the Slack poll earlier we voted for the `@` symbol as the preferred symbol: https://infinitered.slack.com/archives/C0X0P5LP7/p1639524256122600?thread_ts=1639519937.114400&cid=C0X0P5LP7

I don't want to bikeshed too much over this, but I personally have a strong dislike for `~` as to me that indicates my user's home directory, to use it for anything else feels like overloading it

Reply:
Could even get crazy and do the `@/` path alias that NextJs likes: https://nextjs.org/docs/advanced-features/module-path-aliases

<img width="502" alt="Screenshot 2023-03-14 at 3 02 24 PM" src="https://user-images.githubusercontent.com/37849890/225151553-0bb77b35-47a2-4a51-9df7-c6705939d122.png">

Reply:
It looks like from the poll the absolute path alias (`components/card`) and the `@/` alias (`@/components/card`) were split.

Personally, of the two, I would prefer the absolute path alias (`components/card`) because it has had a better failure mode for intellisense when the tsconfig.json isn't indexed or found for some reason.

However, `@/` is nice semantically because it's very clear that it's an alias and distinct from other types of aliases.

Reply:
I believe the two alternatives were `app/components/card` vs `@/components/card`.

Reply:
Right, that is correct

Reply:
> I believe from the Slack poll earlier we voted for the `@` symbol as the preferred symbol: https://infinitered.slack.com/archives/C0X0P5LP7/p1639524256122600?thread_ts=1639519937.114400&cid=C0X0P5LP7
> 
> I don't want to bikeshed too much over this, but I personally have a strong dislike for `~` as to me that indicates my user's home directory, to use it for anything else feels confusing

Oh I was going to say how did I miss this poll, but it was before I was around hah! I get that `~` does look like the user home directory. I'll update it to `@/` I think? just plain `components/card` doesn't given indication in any way that it is an alias

Reply:
I set up the assets directory too seeing as we reference that one quite a bit as well for icons, images

Reply:
Nice addition!